zhaquirks/legrand/wirelessswitch.py

```diff
@@ -0,0 +1,42 @@
+"""Module for Legrand wireless radiant switch."""
+
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.zcl.clusters.general import BinaryInput
+
+from zhaquirks.const import (
+    BUTTON,
+    COMMAND,
+    COMMAND_MOVE,
+    COMMAND_OFF,
+    COMMAND_ON,
+    COMMAND_STOP,
+    LONG_PRESS,
+    LONG_RELEASE,
+    PARAMS,
+    SHORT_PRESS,
+    TURN_OFF,
+    TURN_ON,
+)
+from zhaquirks.legrand import LEGRAND, LegrandPowerConfigurationCluster
+
+(
+    QuirkBuilder(f" {LEGRAND}", " Remote switch")
+    .replaces(LegrandPowerConfigurationCluster)
+    .removes(BinaryInput.cluster_id)
```

Comment:
As suggested above, I'll change the `.removes(BinaryInput.cluster_id)` to just prevent discovery of all default ZHA entities.
```suggestion
    .prevent_default_entity_creation(endpoint_id=1, cluster_id=BinaryInput.cluster_id)
```

Reply:
Thank you!